Automattic/jetpack
# VideoPress: filter videos by Filter Section

This PR connects the Filter section with the data handling to filter videos by `privacy` and `rating`.
Disclaimer: there is an issue when filtering by more than one value, Handled by https://github.com/Automattic/jetpack/pull/26830

Follow-up of https://github.com/Automattic/jetpack/pull/26762

Fixes #

#### Changes proposed in this Pull Request:

* VideoPress: implement videos filter section

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to VideoPress dashboard
* Confirm it's possible to filter by `rating` and `privacy`

https://user-images.githubusercontent.com/77539/195735721-6ae0f857-2c2d-44df-a81a-c861ecb0800d.mov
